public/app/plugins/panel/timeseries/config.ts

```diff
@@ -38,6 +38,7 @@ export const defaultGraphConfig: GraphFieldConfig = {
   },
   axisGridShow: true,
   axisCenteredZero: false,
+  axisShow: false,
```

Comment:
Should this be true by default?

Reply:
As I see it's disabled in TimeSeries by default. So probably we have to leave it like that for now.